scipy/spatial/ckdtree.pyx

```diff
@@ -613,6 +608,26 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
             Number of jobs to schedule for parallel processing. If -1 is given
             all processors are used. Default: 1.
                         
+        Attributes
+        ----------
+        data : array_like, shape (n,m)
+            The n data points of dimension m to be indexed. This array is
+            not copied unless this is necessary to produce a contiguous
+            array of doubles, and so modifying this data will result in
+            bogus results. The data are also copied if the kd-tree is built
+            with copy_data=True.
+        leafsize : positive int
+            The number of points at which the algorithm switches over to
+            brute-force. Default: 16.
```

Comment:
remove ``Default: 16``. This makes sense for the description in Parameters, but here it's not needed.